library/Fission/CLI/Command/Up.hs

```diff
@@ -54,7 +55,8 @@ up Up.Options {..} = handleWith_ Error.put' do
   logDebug $ "Starting single IPFS add locally of " <> displayShow absPath
 
   unless dnsOnly do
-    void . liftE . Auth.withAuth $ CLI.Pin.run cid
+    config <- liftE Auth.get
+    void . liftE $ CLI.Pin.run cid (peers config) (toBasicAuth config)
```

Comment:
We do these exact two lines in a couple of places. May be worth DRYing up.

Reply:
Actually, we're doing this because it's no longer just auth in that data structure. I recommend splitting them, as per another comment, and then we can go back to `withAuth`.